src/main/webapp/style.css

```diff
@@ -202,11 +203,11 @@ header {
   width: calc(50% - 60px);
 }
 
-li.pointer {
+.pointer {
```

Comment:
I just noticed - although the topic is one of debate (see atomic CSS), I would advocate against a `pointer` class here. Instead, I'd opt to give the li a descriptive class name and styling that with the cursor: pointer style.

Reply:
I was thinking of that too. For the hover, is there a styling name that would also go within this class styling or whould i still have to do like li:hover {} ?